files/initramfs-tools/arista-net

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
```

Comment:
add comment for this file 

rename the management interfaces on arista switches